src/main/java/org/reaktivity/nukleus/http/internal/routable/stream/ServerAcceptReplyState.java

```diff
@@ -19,36 +19,34 @@
 
 import org.reaktivity.nukleus.http.internal.routable.Target;
 
-final class ServerConnectionState
+final class ServerAcceptReplyState
```

Comment:
I have renamed to ServerAcceptState. I want to retain the "accept" part because it indicates this is state for the low level side, as opposed to "server connect" which is the high level side.